OpenScholar/azuredeploy.json

```diff
@@ -189,7 +202,7 @@
         "autoUpgradeMinorVersion": true,
         "settings": {
           "fileUris": [
-            "[variables('installScriptUri')]"
+            "[concat(parameters('_artifactsLocation'), '/', variables('installScriptName'), parameters('_artifactsLocationSasToken'))]"
```

Comment:
use the uri function and remove the "/" - that will break if someone supplies a trailing slash on the location, e.g.

uri(parameters('_artifactsLocation'), concat(variables('installScriptName'), parameters('_artifactsLocationSasToken')))